hack/jenkins/common.sh

```diff
@@ -39,6 +41,8 @@ echo "uptime:    $(uptime)"
 # Setting KUBECONFIG prevents the version ceck from erroring out due to permission issues
 echo "kubectl:   $(env KUBECONFIG=${TEST_HOME} kubectl version --client --short=true)"
 echo "docker:    $(docker version --format '{{ .Client.Version }}')"
+echo "go:        $(/usr/local/go/bin/go version || true)"
```

Comment:
Can this just be `go version`, where /usr/local/go/bin is the first item in path?